projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/methods-and-properties.migration.ts

```diff
@@ -0,0 +1,1790 @@
+      importPath: '@spartacus/cart/quick-order/core',
+      deprecatedNode: 'removeEntry',
+      comment:
+        "// TODO:Spartacus - Method 'removeEntry' was removed from Class 'QuickOrderService'. Use 'softDeleteEntry' instead.",
+    },
+    {
+      class: 'QuickOrderService',
+      importPath: '@spartacus/cart/quick-order/core',
+      deprecatedNode: 'search',
+      comment:
+        "// TODO:Spartacus - Method 'search' was removed from Class 'QuickOrderService'. Use 'searchProducts' instead.",
+    },
+    {
+      class: 'QuickOrderFacade',
+      importPath: '@spartacus/cart/quick-order/root',
+      deprecatedNode: 'removeEntry',
+      comment:
+        "// TODO:Spartacus - Method 'removeEntry' was removed from Class 'QuickOrderFacade'. ",
+    },
+    {
+      class: 'QuickOrderFacade',
+      importPath: '@spartacus/cart/quick-order/root',
+      deprecatedNode: 'search',
+      comment:
+        "// TODO:Spartacus - Method 'search' was removed from Class 'QuickOrderFacade'. ",
+    },
+    {
+      class: 'AddToSavedCartComponent',
+      importPath: '@spartacus/cart/saved-cart/components',
+      deprecatedNode: 'activeCartService',
+      comment:
+        "// TODO:Spartacus - Property 'activeCartService' was removed from Class 'AddToSavedCartComponent'. ",
+    },
+    {
+      class: 'SavedCartAdapter',
+      importPath: '@spartacus/cart/saved-cart/core',
+      deprecatedNode: 'saveCart',
+      comment:
+        "// TODO:Spartacus - Method 'saveCart' was removed from Class 'SavedCartAdapter'. ",
+    },
+    {
+      class: 'SavedCartConnector',
+      importPath: '@spartacus/cart/saved-cart/core',
+      deprecatedNode: 'saveCart',
+      comment:
+        "// TODO:Spartacus - Method 'saveCart' was removed from Class 'SavedCartConnector'. ",
+    },
+    {
+      class: 'SavedCartEventBuilder',
+      importPath: '@spartacus/cart/saved-cart/core',
+      deprecatedNode: 'registerDeleteSavedCartEvents',
+      comment:
+        "// TODO:Spartacus - Method 'registerDeleteSavedCartEvents' was removed from Class 'SavedCartEventBuilder'. It was moved to 'CartEventBuilder', and was renamed to 'registerDeleteCart'.",
+    },
+    {
+      class: 'OccSavedCartAdapter',
+      importPath: '@spartacus/cart/saved-cart/occ',
+      deprecatedNode: 'getSaveCartEndpoint',
+      comment:
+        "// TODO:Spartacus - Method 'getSaveCartEndpoint' was removed from Class 'OccSavedCartAdapter'. ",
+    },
+    {
+      class: 'OccSavedCartAdapter',
+      importPath: '@spartacus/cart/saved-cart/occ',
+      deprecatedNode: 'saveCart',
+      comment:
+        "// TODO:Spartacus - Method 'saveCart' was removed from Class 'OccSavedCartAdapter'. ",
+    },
+    {
+      class: 'NewSavedCartOrderEntriesContext',
+      importPath: '@spartacus/cart/saved-cart/root',
+      deprecatedNode: 'actionsSubject',
+      comment:
+        "// TODO:Spartacus - Property 'actionsSubject' was removed from Class 'NewSavedCartOrderEntriesContext'. ",
```

Comment:
Out of many cases, this is an example of the case I was talking about, where a property getting deleted due to the dependency to `ActionsSubject` being removed (which is good) https://github.com/SAP/spartacus/pull/15307#discussion_r830746461

However, the case I am mentioning is about also including in the migration saying `importInfoService` was added since a new dependency was introduced `ProductImportInfoService`

Reply:
So it would be `// TODO:Spartacus - Property 'importInfoService' was added from Class 'NewSavedCartOrderEntriesContext'.`

Although, thinking about it now, it does not make sense mentioning these properties as they're not important in a constructor. The variable property can be named anything. 

All in all, should we mention constructor properties in the migrations?